task1_materials_castelar/models/stock_move.py

```diff
@@ -0,0 +1,17 @@
+from odoo import api, models
+from odoo.exceptions import UserError
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+    
+    @api.onchange('product_uom_qty','quantity_done')
```

Comment:
While this is possible, @api.contrains is preferable in this situation as it's meant to trigger error messages based on certain conditions.